src/main/java/org/jenkinsci/plugins/workflow/graphanalysis/ChunkFinder.java

```diff
@@ -0,0 +1,34 @@
+package org.jenkinsci.plugins.workflow.graphanalysis;
+
+import org.jenkinsci.plugins.workflow.graph.FlowNode;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+
+/**
+ * Matches the start and end of a chunk
```

Comment:
I am guessing this type is just a helper that you pass to something else that is better documented—so link to that Javadoc in case someone is unlucky enough to start browsing here.